helper-cli/src/main/kotlin/commands/GenerateRuleViolationResolutionsCommand.kt

```diff
@@ -58,16 +58,18 @@ internal class GenerateRuleViolationResolutionsCommand : CliktCommand(
 
     private val severity by option(
         "--severity",
-        help = "Only consider violations of the given severity. Allowed values: ERROR|WARNING|HINT."
-    ).enum<Severity>()
+        help = "Only consider violations of the given severity specified as comma separated values. Allowed values: " +
```

Comment:
"...given severities", and mabye add a comma after that word.
Maybe also rename the property and option to `severities`?

Reply:
I've updated the comment and kept `severity`.